app/controllers/projects_controller.rb

```diff
@@ -40,19 +42,22 @@ def edit
   def update
     if @project.update(project_params)
       flash[:success] = t('flash_messages.update', name: t('shared.project'))
+      respond_to do |format|
+        format.js { redirect_to projects_path }
+      end
     else
-      flash[:danger] = t('flash_messages.error', error_msg: @project.errors.full_messages.first)
-    end
-    respond_to do |format|
-      format.js { redirect_to projects_path }
+      respond_to do |format|
+        format.js
+      end
     end
   end
 
   def show
     add_breadcrumb @project.name, :project_path
-    @pagy, @project_issues = pagy(@project.issues, items: Company::PAGE_SIZE)
+    @pagy, @project_issues = pagy(@project.issues, link_extra: "data-remote='true'", items: Company::PAGE_SIZE)
```

Comment:
If this is to be used everywhere, move to pagy.rb

Reply:
will do in next PR